docs/getting_started/tutorial.md

```diff
@@ -306,7 +317,12 @@ In the model above, we import `index` as this makes the model searchable. You ca
 
 Run `python manage.py makemigrations` and `python manage.py migrate`.
 
-Create a template at `blog/templates/blog/blog_page.html`:
+Create a template by following these steps:
+1. Goto your `blog` folder.
+2. Goto the `templates` folder in your `blog` folder.
+3. Goto the `blog` folder in your `templates` folder.
+4. Create a file named named `blog_page.html` in the `blog` folder mentioned in step 3.
+5. Add the following content to your newly created `blog_page.html` file:
```

Comment:
Similar here, see if we can condense this into a bit more detail than exists now but without hand holding the entire process of creating a folder each step of the way.


```suggestion
Create a new template file at the location `blog/templates/blog/blog_page.html`, you may need to create the folders within your `blog` app's folder. 

Now add the following content to your newly created `blog_page.html` file:
```